bokeh/charts/__init__.py

```diff
@@ -1,16 +1,16 @@
 
-from .builder.area_builder import Area
-from .builder.donut_builder import Donut
-from .builder.dot_builder import Dot
-from .builder.line_builder import Line
-from .builder.step_builder import Step
-from .builder.histogram_builder import Histogram
-from .builder.bar_builder import Bar
-from .builder.scatter_builder import Scatter
-from .builder.boxplot_builder import BoxPlot
-from .builder.timeseries_builder import TimeSeries
-from .builder.heatmap_builder import HeatMap
-from .builder.horizon_builder import Horizon
+from .builder.area_builder import Area  # noqa
+from .builder.donut_builder import Donut  # noqa
+from .builder.dot_builder import Dot  # noqa
+from .builder.line_builder import Line  # noqa
+from .builder.step_builder import Step  # noqa
+from .builder.histogram_builder import Histogram  # noqa
+from .builder.bar_builder import Bar  # noqa
+from .builder.scatter_builder import Scatter  # noqa
+from .builder.boxplot_builder import BoxPlot  # noqa
+from .builder.timeseries_builder import TimeSeries  # noqa
+from .builder.heatmap_builder import HeatMap  # noqa
+from .builder.horizon_builder import Horizon  # noqa
```

Comment:
I don't like splitting the transitional imports like that. I prefer using the name rather than the comment for a couple of reasons, but only right together on the same line as the import:
```
import foo; foo
import bar; bar
```